lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -635,6 +663,27 @@ dummy 1.2.3
             depName: 'shfmt',
             extractVersion: '^v(?<version>\\S+)',
           },
+          {
+            currentValue: '2.14.0',
+            datasource: 'github-tags',
+            packageName: 'GoogleContainerTools/skaffold',
+            depName: 'skaffold',
+            extractVersion: '^v(?<version>\\S+)',
+          },
+          {
+            currentValue: '3.0.18',
+            datasource: 'github-tags',
+            packageName: 'budimanjojo/talhelper',
+            depName: 'talhelper',
+            extractVersion: '^v(?<version>\\S+)',
+          },
+          {
+            currentValue: '1.9.3',
+            datasource: 'github-tags',
```

Comment:
Why tags and not releases?

Reply:
I'm matching what's used by the ASDF plugin itself:
- https://github.com/bjw-s/asdf-talosctl/blob/main/bin/list-all#L11 uses `list_all_versions()`
- `list_all_versions()` calls `list_github_tags()` in https://github.com/bjw-s/asdf-talosctl/blob/main/lib/utils.bash#L52-L61